src/decide.js

```diff
@@ -0,0 +1,56 @@
+import { autocapture } from './autocapture'
+import { _ } from './utils'
+
+const decide = {
```

Comment:
Code style nit: Let's use classes as with `src/extensions/sessionrecording.js`.

So constrctor + `callDecide` function.

Reply:
Ack had it as a class first but thought it was more straightforward like this, reverting to class 😉